Library/Formula/cyassl.rb

```diff
@@ -55,6 +67,8 @@ def install
       args << "--disable-fastmath" << "--disable-fasthugemath"
     end
 
+    args << "--enable-aesni" if Hardware::CPU.aes?
```

Comment:
Oh wait, I'm an idiot. An exclamation mark at the start of something usually means ` if not `. Apologies for the stupid question.